core/src/test/java/io/trinitylake/TestObjectDefinitions.java

```
@@ -43,12 +43,7 @@ public class TestObjectDefinitions {
   private final ViewDef testViewDef =
       ObjectDefinitions.newViewDefBuilder()
           .setSchemaBinding(false)
-          .addSqlRepresentations(
-              SQLRepresentation.newBuilder()
-                  .setType("sql")
-                  .setSql("select 'foo' foo")
-                  .setDialect("spark-sql")
-                  .build())
+          .setSubstraitPlan(ByteString.copyFromUtf8("substrait plan"))
```

We should use an actual plan instead of just string. We should actually throw a failure at TrinityLake class level if it fails to deserialize back to a Substrait plan